grammar.coffee

```diff
@@ -195,6 +205,14 @@ module.exports = grammar
 			choice(@_loop_statement, @if_statement)
 		)
 
+		break_statement: -> prec(PREC.BREAK_STATEMENT, seq('break', optional(@identifier)))
+
+		continue_statement: -> prec(PREC.CONTINUE_STATEMENT, seq('continue', optional(@identifier)))
+
+		fallthrough_statement: -> 'fallthrough'
+
+		return_statement: -> prec(PREC.RETURN_STATEMENT, seq('return', optional(@_expression)))
```

Comment:
@maxbrunsfeld: I can’t seem to make this work. Given:

```
return
return x
```

the parser seems to always prefer to parse that as `return` followed by a pair of identifiers (which shouldn’t be a valid production at all).